pep-0677.rst

```diff
@@ -655,44 +655,47 @@ Our plan is to expose a structured API as if they were defined as follows::
     class CallableType:
         is_async: bool
         arguments: Ellipsis | tuple[CallableTypeArgument]
-        return_type:: Typing.type
+        return_type
 
     class CallableTypeArgument:
         kind: CallableTypeArgumentKind
-        annotation: typing.Type | typing.TypeVar | typing.ParamSpec
+        annotation
```

Comment:
That's not valid syntax to declare a variable -- you have to add `: object`.

Reply:
Gaah. This is what happens as soon as I don't have a linter or typechecker yelling at me haha